### PR TITLE
add-projectlive-nf

### PR DESCRIPTION
**Pull request by jaybee84**
This is a test, and the concerned repos are still work in progress.

ProjectLive-NF backend data files are being generated using a docker image built using [this repo](https://github.com/Sage-Bionetworks/projectlive-nf-data) and stored on dockerhub/sagebio.

In an effort to consolidate where projectlive-nf related things live, I have copied the contents of the above repo in the form of a folder inside this repo and added a yml file into the .github/workflows/ directory.

We might want to use the ghcr.io registry instead of the dockerhub registry for projectLive-NF like all other NF related jobs.

Adding this as a draft PR, will finalize after discussions with appropriate stakeholders.

### CODE REVIEW

**Comment by jaybee84** [comment]
There is a newer version of the docekerfile pull from projectlive-nf-data repo again before pushing this PR

File: projectlive-nf-datafiles/R/create_rds_files.R [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+live_folder <- "syn22281727"
+dev_folder  <- "syn24474593"
+
+source("https://raw.githubusercontent.com/Sage-Bionetworks/projectlive.modules/231f14aa9a4c35a4cad46c851ff05eb07dff3f19/R/synapse_functions.R")
+source("https://raw.githubusercontent.com/Sage-Bionetworks/projectlive.modules/231f14aa9a4c35a4cad46c851ff05eb07dff3f19/R/data_manipulation_functions.R")
+
+reticulate::use_condaenv("sage-bionetworks", required = T)
+synapseclient <- reticulate::import("synapseclient")
+syn <- synapseclient$Synapse()
+auth_token <- rjson::fromJSON(Sys.getenv("SCHEDULED_JOB_SECRETS"))$SYNAPSE_AUTH_TOKEN
```
Comment (andrewelamb) [inline projectlive-nf-datafiles/R/create_rds_files.R:12]: This needs to be changed to: `auth_token <- "${{ secrets.SYNAPSE_AUTH_TOKEN }}",`

File: projectlive-nf-datafiles/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# projectlive-nf-data
+Builds a docker image for data processing for the NF version of projectLive
+[App repo](https://github.com/Sage-Bionetworks/projectLive_NF)
```
Comment (anngvu) [inline projectlive-nf-datafiles/README.md:3]: Maybe for consistency, note that only SYNAPSE_AUTH_TOKEN is needed for this job in a section like https://github.com/nf-osi/jobs/tree/main/snapshot#secrets-and-env-vars

Comment (jaybee84) [inline projectlive-nf-datafiles/README.md:3]:
```suggestion
[App repo](https://github.com/Sage-Bionetworks/projectLive_NF)

### Secrets and env vars

```
SCHEDULED_JOB_SECRETS={"SYNAPSE_AUTH_TOKEN":"xxxxxxxxxxxxxxxxxxxxxxxxxxxx"}
LABEL=projectlive-nf-data
SCHEDULE=daily
COMMENT="Scheduled projectlive-nf-data"
SLACK=https://hooks.slack.com/services/xxxxxxxxxxxxxxxxxxxxxxxxxxxx
```

### Example run 

`docker run --env-file envfile run ghcr.io/nf-osi/jobs-snapshot syn27242487 syn27242485`
```

File: projectlive-nf-datafiles/README.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# projectlive-nf-data
+Builds a docker image for data processing for the NF version of projectLive
```
Comment (jaybee84) [inline projectlive-nf-datafiles/README.md:2]:
```suggestion
Builds a docker image for data processing for the NF version of projectLive
This image is used in a scheduled job to update backend data files powering projectLive-NF. 
```

File: projectlive-nf-datafiles/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,45 @@
+FROM rocker/tidyverse:4.1.0
```
Comment (jaybee84) [inline projectlive-nf-datafiles/Dockerfile:1]:
```suggestion
FROM rocker/tidyverse:4.1.2
```

**Pull request merged**